src/gas/fetchGasEstimatesViaEthFeeHistory/BlockFeeHistoryDatasetFetcher.ts

```diff
@@ -51,10 +48,6 @@ export default class BlockFeeHistoryDatasetFetcher {
     };
   }
 
-  forLongRange() {
-    return this.fetchExcludingNextBlock({ numberOfBlocks: 20_000 });
-  }
-
   forMediumRange() {
```

Comment:
`forMediumRange` has the same `percentiles` as `forLatestWithNextBlock`. Couldn't you group those into one call to the node instead of two? 🤔

Reply:
By using `fetchIncludingNextBlock` in `forMediumRange`

Reply:
That's a good point. When I first implemented this approach, I thought about that, and for some reason I thought that the data specified by `percentiles` changed based on the block range requested, so even if you kept `percentiles` the same but had a different block range, you might get different results. I realize now that that this may not be the case. And actually — I think we could combine all of these requests into one request.

I want to get this PR merged and released today to relieve the immediate pressure on Infura, but I'm going to work on another PR to test this theory.